### ISSUE TITLE
NullPointerException in LoggingHandler

### ISSUE THREAD

**Issue by mcweba** [swisspush/gateleen#292]
Lately a lot of _NullpointerExceptions_ have been logged. See Stacktrace below

```
java.lang.NullPointerException
	at org.swisspush.gateleen.logging.LoggingHandler.headersAsJson(LoggingHandler.java:347)
	at org.swisspush.gateleen.logging.LoggingHandler.log(LoggingHandler.java:300)
	at org.swisspush.gateleen.logging.LoggingHandler.log(LoggingHandler.java:285)
	at org.swisspush.gateleen.routing.Forwarder.lambda$null$4(Forwarder.java:375)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
	at io.vertx.core.impl.EventLoopContext.lambda$executeAsync$0(EventLoopContext.java:38)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:462)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by mcweba** [comment]
This bug is related to #293

**Comment by mcweba** [comment]
Fixed in version [v1.1.50](https://github.com/swisspush/gateleen/releases/tag/v1.1.50)
